popup.js

```diff
@@ -0,0 +1,170 @@
+const projects = {
+  p15: {
+    name: 'My project 1',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    img: './Images/Desktop-version/nature.png',
+    technolgies: ['React', 'Redux', 'HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://github.com/FaranoshAmini',
+    generalInf: ['Microverse- Student', 'Web Developer', 2021],
+  },
+  p16: {
+    name: 'My project 2',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    img: './Images/Desktop-version/nature.png',
+    technolgies: ['React', 'Redux', 'HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://github.com/FaranoshAmini',
+    generalInf: ['Microverse- Student', 'Web Developer', 2021],
+  },
+  p17: {
+    name: 'My project 3',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    img: './Images/Desktop-version/nature.png',
+    technolgies: ['React', 'Redux', 'HTML', 'CSS', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://github.com/FaranoshAmini',
+    generalInf: ['Microverse- Student', 'Web Developer', 2021],
+  },
+};
+const myWork = document.querySelector('#work');
+function createMobileProjectCard(key) {
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that your popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, tests, margins).

I can see that the colors for your *Microverse- Student* and *Web Developer • 2021* are wrong. please make use of the colors from your design template